tests/objc-cli/test-managed.m

```diff
@@ -22,17 +22,21 @@ int main ()
 		NSLog (@"%s instance property getter", [query answer] == 42 ? "[PASS]" : "[FAIL]");
 		query.answer = 911;
 		NSLog (@"%s instance property setter", [query answer] == 911 ? "[PASS]" : "[FAIL]");
+		[query release];
```

Comment:
I don't think we can _enforce_ ARC in code that consumes the bindings, but this is code we control ourselves, so we can switch to ARC if we want to.